window_doc.talon

```diff
@@ -1,8 +1,7 @@
 os: mac
---
-
-document open: user.open_current_doc()
+-- document open: user.open_current_doc()
```

Comment:
This doesn't seem correct :/

Reply:
Legitimately wondering what this is actually parsing as?

Reply:
I'm on the fence about how to handle this. On the one hand, Talon doesn't support more than one `-` to end the context, so I could see a case that tree-sitter-talon shouldn't go out of its way to support that syntax.

On the other hand, Talon used to support more than one `-` up until probably 6 months ago, so it might make sense to make the parser handle this case, in case people still have that

My temptation would be to just manually replace all the `--` lines with `-` in this repo and then re-run the formatter.  And then can file an issue in `tree-sitter-talon` about supporting `--` for backwards compatibility with older Talon files 🤷‍♂️

Reply:
Left a comment here: https://github.com/wenkokke/tree-sitter-talon/issues/16#issuecomment-1248321204

I think it's fine if `talonfmt` just raises an error (it does is elsewhere if you have invalid rules) so the user is prompted to update their syntax.